process-engine/consumer_api_meta
# ✅ Activate UserTask Tests

_✔︎ waited on https://github.com/process-engine/consumer_api_core/pull/7_

## What did you change?

I added everything necessary to be able to activate our usertask-tests for the consumer_api

## How can others test the changes?

- run `./setup.sh`
- cd into `_integration_tests`
- have a postgres-db setup (see skeleton-database)
- run `npm start`
- wait for the bootstrapper to start
- stop the program
- run npm test

## PR-Checklist

Please check the boxes in this list after submitting your PR:
